packages/wxt/src/core/resolve-config.ts

```diff
@@ -1,4 +1,5 @@
 import { loadConfig } from 'c12';
+import { createRequire } from 'node:module';
```

Comment:
I believe checks are failing because you moved this to a static import. It was causing problems as a static, that's why I used a dynamic import before.

Reply:
The failed check is due to `.cjs` entry points being resolved by `nodeRequire.resolve`. The CJS entry point then uses `require("wxt")` which is not supported, because it resolves to an ES module.

Reply:
My recent commit has fixed this.